### ISSUE TITLE
Sawn-Off Barrels and Pipe Shotguns

### ISSUE THREAD

**Issue by Coaxy** [CleverRaven/Cataclysm-DDA#59815]
### Describe the bug

You can saw off the barrel of a pipe shotgun but the length of the gun does not change as you'd expect.

### Steps to reproduce

1. Get a Pipe Shotgun (Single or Double)
2. Saw off the barrel
3. See that it has gone from 37in to 37in (or 36in for the double-barrel)

### Expected behavior

I expected that the barrel would be shorter. Especially compared to the single barrel shotgun, which is sawed down from 37 inches to 25 inches, the pipe shotgun losing a total of 0 inches seems unintended. 

In addition, for the normal pipe shotgun, volume 2.25L -> 2.21L weight 5.00lbs -> 4.92lbs dispersion 855 -> 930, so it is making a change, just not a very large one.

Having done (only very cursory research) on this in earlier github issues, I see this fix to seemingly the same issue - https://github.com/CleverRaven/Cataclysm-DDA/pull/46927. Perhaps the pipe shotguns did not have their barrel_volume updated alongside the others?

### Versions and configuration

- 0-F.3 (have not confirmed state wrt to experimental)

**Comment by Coaxy** [comment]
I peeked current main branch and it does appear that pipe shotguns missed the barrel_volume change.

The single barrel shotgun (in shot.json) has a barrel_volume of 670ml where the pipe shotgun has a barrel_volume of 36ml (the pipe double barrel has a volume of 72ml, which is presumably why it lost an inch where the single didn't).

I admit to being unfamiliar with mechanics of guns in general (and shotguns in specific). As far as I can tell, the pipe shotguns are not different than the single barreled shotgun in mechanical function and just in quality of manufacture (by the descriptions, at least). I would make the change to their barrel_volume so that it matches the (non-pipe) single/double barrel, but is that reasonable?

Further, I was making sure that none of the other shotgun's barrel_volumes were at (obviously) strange volumes. I came across the "Corbray Streetsweeper" (the revolver shotgun) which also has a barrel_volume of 36ml. The Saiga_12 also only has a barrel volume of 81ml, and the Remington_870_express has a barrel_volume of 20ml.
